### DISCUSSION TITLE
DIfferent src= for image reponsive

### DISCUSSION THREAD

**Discussion by vitalijalbu** [chakra-ui/chakra-ui#8390] [Q&A] [answered]
Ho guys, seems chakra doesn't support it:

```
<Image 
loading="lazy" 
src={{ base: "/images/logo.svg", sm: "/images/logo.svg", md: "/images/logo.svg" }} 
h={{ base: '42px' }} 
className="site-logo" />
```

**Comment by segunadebayo** [comment, marked as answer]
`Image` `src` is a string. A responsive object there is ignored.

Use a native `<picture>` with `<source media="...">`, or pass `srcSet` / `sizes` on the image. Height and width can still be responsive style props.
